src/docs/src/config/intro.rst

```diff
@@ -126,9 +126,10 @@ by at least one space on each side. This might be useful in the ``[jwt_keys]``
 section, where base64 encoded keys may contain some ``=`` characters.
 
 .. note::
-    In case when you'd like to remove some parameter from the `default.ini`
-    without modifying that file, you may override in `local.ini`, but without
-    any value::
+    In case you'd like to remove some parameter from an `.ini` file
+    (e.g. `default.ini`) without modifying that file, you may override
+    it in a subsequently loaded `.ini` file (e.g. `local.ini`), but
```

Comment:
Is the concept of being "subsequently loaded" clarified somewhere in the documentation?  That is, is the order of processing for `.ini` files explicitly discussed somewhere above?

Reply:
Yes, [here](https://docs.couchdb.org/en/stable/config/intro.html#configuration-files) is where the docs discuss loading configuration files in order. I'm very open to improved wording though...

Reply:
Ah, that is perfect, thank you.  I do not think I have problems with the wording.  I just wondered if this has been made explicit up to this point.